mitodl/open-discussions
# Remove zendesk help and replace with "Contact us" email link

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #1187 

#### What's this PR do?
- Removes the zendesk help button
- Adds a 'Contact us' email link

#### How should this be manually tested?
View any page.  There should no longer be a 'Help' overlay button near the the botton right corner.  There should be a 'Contact us' link just above 'Community guidelines' on the left sidebar.  Clicking it should open up a new email addressed to the `OPEN_DISCUSSIONS_SUPPORT_EMAIL` address.

<img width="383" alt="screen shot 2018-11-19 at 3 21 04 pm" src="https://user-images.githubusercontent.com/187676/48732836-c5fa1b80-ec0e-11e8-9a20-7fdc3fc96a76.png">
<img width="367" alt="screen shot 2018-11-19 at 3 20 50 pm" src="https://user-images.githubusercontent.com/187676/48732837-c5fa1b80-ec0e-11e8-8e70-3f3639b5d53c.png">
